crates/forge/tests/integration/gas.rs

```diff
@@ -784,5 +991,14 @@ fn events_contract_cost() {
     // 96 = gas cost of onchain data (deploy cost)
     // 6 gas for 50 event values
     // ~13 gas for 50 event keys
-    assert_gas(&result, "event_emission_cost", 11 + 96 + 6 + 13);
+    // 0 l1_gas + 96 l1_data_gas + (11 + 6 + ~13) * (100 / 0.0025) l2 gas
+    assert_gas(
+        &result,
+        "event_emission_cost",
+        GasVector {
+            l1_gas: GasAmount(0),
+            l1_data_gas: GasAmount(96),
+            l2_gas: GasAmount(1_208_000),
```

Comment:
How does this work exactly? It's because it's `~30 * 40,000` not `30 * 40,000`?

Reply:
The same as https://github.com/foundry-rs/starknet-foundry/pull/3054#discussion_r1987609375

the event does not cost exactly 13, but 13.2 (so your comments here were an approximation, I beliebe that is why someone said ~13 and not 13), hence discrepancy. Again: I did not verify those number, just assumed You did a good job doing it (I just did a conversion, which does not introduce any measuring errors). I only verified sierra gas calculations.

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/3054#discussion_r1989865883